mediachain/indexer/mc_simpleclient.py

```diff
@@ -209,11 +209,11 @@ class SimpleClient(object):
     
     def __init__(self,
                  ## Hard Code for now:
-                 datastore_host = '107.23.23.184',
+                 datastore_host = 'facade.mediachain.io',
                  datastore_port = 10002,
-                 transactor_host = '107.23.23.184',
+                 transactor_host = 'facade.mediachain.io',
```

Comment:
we should make this configurable, so that we can use the internal facade IP when running in our vpc.